pkg/etcd/client_for_api.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcd
+
+import (
+	"context"
+
+	"github.com/pingcap/tiflow/cdc/model"
+	"go.etcd.io/etcd/api/v3/mvccpb"
+)
+
+// CDCEtcdClientForAPI extracts CDCEtcdClients's method used for apiv2
+// TODO: refactor CDCEtcdClient as interface, and use it for v1
+type CDCEtcdClientForAPI interface {
```

Comment:
Why not just name it `CDCEtcdClient`?

Reply:
here we only mock a part of `CDCEtcdClient` functions for API, later we will add all method an then rename this interface.